desktop/cmp/button/AppMenuButton.js

```diff
@@ -95,15 +101,18 @@ function buildMenuItems({
     hideChangelogItem,
     hideFeedbackItem,
     hideImpersonateItem,
+    hideInspectorItem,
```

Comment:
I might be happy to leave this out in favor of a more obscure keyboard shortcut.  Let's discuss.  This to me seems like more of a developer's tool (unlike e.g. admin/impersonation which really are for business users as well).  Not sure I want to make promises that it is "part" of the app...